prompt/prompt-executor/prompt-executor-clients/prompt-executor-google-client/src/commonMain/kotlin/ai/koog/prompt/executor/clients/google/GoogleLLMClient.kt

```diff
@@ -646,6 +646,28 @@ public open class GoogleLLMClient(
                         )
                     )
 
+                    is GooglePart.InlineData -> {
+                        val inlineData = part.inlineData
+                        val contentPart = when (val mimeType = inlineData.mimeType) {
+                            "image/png", "image/jpeg", "image/webp" -> ContentPart.Image(
+                                content = AttachmentContent.Binary.Bytes(inlineData.data),
+                                format = mimeType.substringAfter("image/"),
+                            )
```

Comment:
Let's keep `mimeType` as it is to avoid "magical" transformations 

```kotlin
@Serializable
    public data class Image(
        override val content: AttachmentContent,
        override val format: String,
        override val mimeType: String = "image/$format", // 🦄 TooManyMagic
        override val fileName: String? = null,
    ) : Attachment {
````
 
```suggestion
                            "image/png", "image/jpeg", "image/webp" -> ContentPart.Image(
                                content = AttachmentContent.Binary.Bytes(inlineData.data),
                                format = mimeType.substringAfter("image/"),
                                mimeType = mimeType,
                            )
```